packages/fluentui/react-bindings/src/hooks/useStyles.ts

```diff
@@ -67,27 +53,11 @@ type InlineStyleProps<StyleProps> = {
   variables?: ComponentVariablesInput;
 };
 
-export const defaultPerformanceFlags: StylesContextPerformance = {
-  enableSanitizeCssPlugin: process.env.NODE_ENV !== 'production',
-  enableStylesCaching: true,
-  enableVariablesCaching: true,
-  enableBooleanVariablesCaching: false,
-};
-
-const defaultContext: StylesContextValue = {
-  rtl: false,
-  disableAnimations: false,
-  performance: defaultPerformanceFlags,
-  renderer: noopRenderer,
-  theme: emptyTheme,
-  telemetry: undefined,
-};
-
 export const useStyles = <StyleProps extends PrimitiveProps>(
   displayName: string,
   options: UseStylesOptions<StyleProps>,
 ): UseStylesResult => {
-  const context: StylesContextValue = React.useContext(ThemeContext) || defaultContext;
+  const context = useFluentContext();
```

Comment:
```suggestion
  const context = useFluentContext() || defaultContext;
```

?

Reply:
Nope, it's not a case anymore. Previously we had this because we need to have a default value, but it was impossible to define it as `.createContext()` was called in `react-fela`. But now we are defining it under own `.createContext()` call:

```tsx
const FluentContext = React.createContext<ProviderContextPrepared>(defaultContext);
```